### ISSUE TITLE
Disable attribute combined with autocomplete

### ISSUE THREAD

**Issue by pacoma93** [Gbuomprisco/ngx-chips#636]
PLEASE MAKE SURE THAT:
- you searched similar issues online (9/10 issues in this repo are solved by googling, so please...do it)
- you provide an online demo I can see without having to replicate your environment
- you help me by debugging your issue, and if you can't, do go on filling out this form

**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[ ] support request/question

Notice: feature requests will be ignored, submit a PR if you'd like
```

**Current behavior**

my <tag-input> contains a autocomplete. my app has a read mode and edit mode. the tag also contains the disable for this. for some reason, i cant create new tags when i have a disable attribute false and a autocomplete.

**Expected behavior**

Im expecting it to create the tag when im in edit mode, and also show my dropdown. 

**Minimal reproduction of the problem with instructions (if applicable)**

**What do you use to build your app?. Please specify the version**

angular CLI: 1.5.5
"@agm/core": "^1.0.0-beta.2",
    "@angular/animations": "^5.0.5",
    "@angular/cdk": "github:angular/cdk-builds",
    "@angular/common": "^5.0.5",
    "@angular/compiler": "^5.0.5",
    "@angular/core": "^5.0.5",
    "@angular/forms": "^5.0.5",
    "@angular/http": "^5.0.5",
    "@angular/material": "github:angular/material2-builds",
    "@angular/material-moment-adapter": "^5.0.0-rc.2",
    "@angular/platform-browser": "^5.0.5",
    "@angular/platform-browser-dynamic": "^5.0.5",
    "@angular/platform-server": "^5.0.5",
    "@angular/router": "^5.0.5",

**Angular version:** 

**ngx-chips version:**

**Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

**Comment by zyborc** [comment]
Hello i have the same issue.

**Comment by Gbuomprisco** [comment]
this should be fixed with version 1.6.8
